ForumProject/projects/urls.py

```diff
@@ -8,5 +8,25 @@
 router.register('', views.ProjectViewSet)
 
 urlpatterns = [
-    path('', include(router.urls))
+    path('', include(router.urls)), 
+    path('files-of-project/<int:project>/', views.ProjectFilesViewSet.as_view({
+        'get': 'list',
+        'post': 'create',
+        'delete': 'destroy',
+    }), name='project_files_by_project'),
+    path('delete-file/<int:projectfiles_id>/of-project/<int:project>/',
+        views.delete_project_file, name='delete_project_file'),
+    path('shortlist-project/<int:project_id>/by-investor/<int:investor_id>/',
+        views.shortlist_project, name='shortlist_project'),
+    path('subscribe-for-project/<int:project_id>/by-investor/<int:investor_id>/for-share/<int:share>/',
+        views.subscribe_for_project, name='subscribe_for_project'),
+    path('delist-project/<int:project_id>/by-investor/<int:investor_id>/',
+        views.delist_project, name='delist_project'),
```

Comment:
Consider renaming URL path segments to use dashes instead of underscores for better readability and consistency with common web practices (e.g., `delete-file` is good, `subscribe-for-project` is consistent, but `delist_project` should be `delist-project`)

Reply:
Renamings have been made